File: MyClassLibrary/Forms/SplashScreen1.cs
using DevExpress.XtraSplashScreen;
using System;

namespace MyClassLibrary.Forms
{
	public partial class SplashScreen1 : SplashScreen
	{
		public SplashScreen1()
		{
			InitializeComponent();
			this.labelCopyright.Text = $"Copyright © 1998-{DateTime.Now.Year}";
		}

		#region Overrides

		public override void ProcessCommand(Enum cmd, object arg)
		{
			base.ProcessCommand(cmd, arg);
		}

		#endregion Overrides

		public enum SplashScreenCommand
		{
		}
	}
}

File: MyClassLibrary/Forms/SplashScreen1.vb
Public Class SplashScreen1

    Sub New()
        InitializeComponent()
    End Sub

    Public Overrides Sub ProcessCommand(ByVal cmd As System.Enum, ByVal arg As Object)
        MyBase.ProcessCommand(cmd, arg)
    End Sub

    Private Sub SplashScreen1_Load(sender As Object, e As System.EventArgs) Handles MyBase.Load
        label_ProductVersion.Text = $"{Application.ProductName} {Application.ProductVersion}"
    End Sub

End Class